Solutions/Apache Log4j Vulnerability Detection/Playbooks/BatchImport/BatchImportToSentinel.json

```diff
@@ -1,6 +1,31 @@
 {
     "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
     "contentVersion": "1.0.0.0",
+    "metadata": {
+        "title": "BatchImportToSentinel", 
+        "description": "These playbooks automate the ingest of threat indicators into the ThreatIntelligenceIndicator table of an Microsoft Sentinel workspace. Sample data for Log4j IOC can be found at https://raw.githubusercontent.com/Azure/AzureSentinel/master/Sample%20Data/Feeds/Log4j_IOC_List.csv.",
+        "prerequisites": ["1. You must deploy the BatchImportToSentinel playbook before deploying the Log4jIndicatorProcessor playbook. \n\n 2. During the deployment of Log4jIndicatorProcessor playbook, Playbook2 parameter name should be that of BatchImportToSentinel(default)."],
```

Comment:
@v-amolpatil , prerequisite for this playbook is not matching, can be empty

Reply:
done